input/data/2015/2015A%26A...574A.100H/tev-000098.yaml

```diff
@@ -5,8 +5,9 @@ telescope: hess
 data:
   livetime: 113h
   significance: 6.6
+  excess: 163
 
-pos:
+pos: #Check
```

Comment:
Remove this `#Check` comment; at least to me it's not clear to me what that means.

Reply:
Ah, that was only for myself to mark that I reviewed the position against the paper.